packages/react/src/utilities/ThemeProvider/ThemeProvider.tsx

```diff
@@ -22,9 +22,6 @@ export const ThemeProvider: React.FunctionComponent<ThemeProviderProps> = React.
   // Render styles.
   useThemeProviderClasses(state);
 
-  // Apply focus rect class on key presses.
-  useFocusRects(state.ref);
-
```

Comment:
This is removed because it was actually never needed in this component, since it has no styles relying on `isFocusVisible` or `isFocusHidden`.